patches/node/chore_remove_errant_semicolons.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Mon, 9 Jan 2023 13:53:36 +0100
+Subject: chore: remove errant semicolons
```

Comment:
Is this patch really necessary -- does this trigger off a linter error on our side or something?

Reply:
it fails compilation owing to Chromium rules - we've had this happen a few times historically